cmd/daprd/options/options.go

```diff
@@ -163,6 +164,7 @@ func New(origArgs []string) (*Options, error) {
 	fs.IntVar(&opts.AppHealthProbeTimeout, "app-health-probe-timeout", int(config.AppHealthConfigDefaultProbeTimeout/time.Millisecond), "Timeout for app health probes in milliseconds")
 	fs.IntVar(&opts.AppHealthThreshold, "app-health-threshold", int(config.AppHealthConfigDefaultThreshold), "Number of consecutive failures for the app to be considered unhealthy")
 	fs.StringVar(&opts.AppChannelAddress, "app-channel-address", runtime.DefaultChannelAddress, "The network address the application listens on")
+	fs.BoolVar(&opts.EnableWorkflowEnhancedLogging, "enable-wf-enhanced-logging", false, "Enable enhanced logging for workflows")
```

Comment:
Please rename all occurrences of "enhanced" to "verbose".

Please mark this flag as deprecated, and add a description that this flag is both temporary, as well as the impact enabling this flag will have.